sql/engines/elasticsearch.py

```diff
@@ -683,7 +680,7 @@ def execute_check(self, db_name=None, sql=""):
                     )
             elif doc.api_endpoint == "_update_by_query":
                 if doc.method == "POST":
-                    if is_pass == False:
+                    if is_pass is False:
                         is_pass = True
```

Comment:
```suggestion
                    is_pass = True
```
这个判断在我看来没有意义, 你看看这样简单写行不行